### PR TITLE
chore: add ethereum vault definitions

### PR DESCRIPTION
**Pull request by axejintao**
# Summary

Register Ethereum vaults

## Tracking

- [x] Signed off by Marcom

Partial of #365

### CODE REVIEW

File: scripts/issue/365/register_vaults.py [hunk 1]
```diff
@@ -189,8 +189,193 @@
...
+  [
+    '0xBE08Ef12e4a553666291E9fFC24fCCFd354F2Dd2',
+    'v1',
+    'name=Tricrypto,protocol=Convex,behavior=None',
+    0
+  ],
+  [
+    '0x2B5455aac8d64C14786c3a29858E43b5945819C0',
+    'v1',
+    'name=cvxCRV,protocol=Convex,behavior=None',
```
Comment (sajanrajdev) [inline scripts/issue/365/register_vaults.py:304]: Shouldn't this be a Ecosystem Helper?

**Comment by sajanrajdev** [comment]
Other than the comments, addresses, metadata and statuses LGTM

File: scripts/issue/365/register_vaults.py [hunk 2]
```diff
@@ -189,8 +189,193 @@
...
+  [
+    '0xfd05D3C7fe2924020620A8bE4961bBaA747e6305',
+    'v1',
+    'name=bveCVX,protocol=Convex,behavior=None',
+    3
+  ],
+  [
+    '0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551',
+    'v1',
+    'name=CVX%20/%20bveCVX,protocol=Curve,behavior=None',
```
[the remark this answers is not in the document]
Comment (axejintao) [reply scripts/issue/365/register_vaults.py:340]:
```
> decodeURI('CVX%20/%20bveCVX')
'CVX / bveCVX'
```

File: scripts/issue/365/register_vaults.py (hunk 1 above)
Comment (axejintao) [reply scripts/issue/365/register_vaults.py:304]: i guess it is

**Comment by axejintao** [comment]
@sajanrajdev 

- updated bcvxCrv behavior to `Ecosystem Helper`
- removed `X / Y` notation and conformation on other vaults to `X/Y`

File: scripts/issue/365/register_vaults.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (axejintao) [reply scripts/issue/365/register_vaults.py]: looks like this is just how it is in the api, i'll update it to conform

File: scripts/issue/365/register_vaults.py [hunk 3]
```diff
@@ -189,8 +189,193 @@
...
+    '0x6aF7377b5009d7d154F36FE9e235aE1DA27Aea22',
+    'v1',
+    'name=remBadger,protocol=Badger,behavior=None',
+    3
+  ],
+  [
+    '0xeC1c717A3b02582A4Aa2275260C583095536b613',
+    'v1',
+    'name=BADGER/WBTC,protocol=Convex,behavior=None',
+    0
```
Comment (jorijnsmit) [inline scripts/issue/365/register_vaults.py:371]: why is status code here `0`?

Comment (axejintao) [reply scripts/issue/365/register_vaults.py:371]: thanks, this was because it was 'featured' i've updated the gen script example for this.

**Review by petrovska-petro — APPROVED** [review]

**Pull request merged**